lib/src/base-styles/_nx-lists.scss

```diff
@@ -128,109 +120,65 @@
  * Action or Clickable Lists
  */
 .nx-list--clickable {
-  .nx-list__title {
-    color: $nx-text-color;
-  }
-
   .nx-list__item {
-    @include semi-bold();
-
-    color: $nx-text-color-light;
-
     &.selected {
-      background-color: $nx-grey-3;
-      color: $nx-text-color;
-      pointer-events: none;
-    }
+      background-color: #d3e6ff;
 
-    &:not(.nx-list__item--empty):not(.nx-disabled):not([disabled]) {
-      &:hover, &:focus {
-        background-color: $nx-color-blue-96;
-        border-top: $nx-border-row-hover;
-        border-bottom: $nx-border-row-hover;
-        color: $nx-text-color;
-        cursor: pointer;
-        outline-style: none; // removes :focus outline
-
-        .fa-chevron-right {
-          color: $nx-color-blue-41;
-        }
-      }
-
-      &:active {
-        background-color: $nx-grey-1;
-        border-top: $nx-border-row-hover;
-        border-bottom: $nx-border-row-hover;
+      &:hover {
+        background-color: #d3e6ff;
       }
     }
 
-    &.nx-list__item--empty {
-      @include regular();
-
-      border-top: $nx-border; // assume it's always the first item
-      color: $nx-text-color-light;
+    &:hover {
+      background-color: #f1f3fa;
+      cursor: pointer;
+      outline-style: none; // removes :focus outline
     }
 
-    &.nx-error {
-      color: $nx-text-color-error;
+    &:active {
+      background-color: #d3e6ff;
+    }
 
-      &:not(.nx-disabled):not([disabled]) {
-        border: $nx-border-error;
 
-        &:hover, &:focus, &:active {
-          background-color: $nx-fail-color-bg;
-          border: $nx-border-error;
-          color: $nx-text-color-error;
-          cursor: default;
-        }
-      }
-    }
+    &.nx-list__item--empty {
+      @include regular();
 
-    .nx-list__subtext {
-      pointer-events: none; // Fix for IE11 :active state display
+      color: $nx-text-color-light;
     }
 
-    &.nx-list__item--disabled {
+    &.disabled {
       color: $nx-text-color-disabled;
-      cursor: default;
 
       &:hover {
-        border-top: 1px solid transparent;
-        border-bottom: $nx-border-row;
+        background-color: transparent;
+        cursor: default;
+      }
+
+      > .nx-chevron {
+        display: none;
       }
     }
-  }
 
-  .fa-chevron-right {
-    margin-right: 3px;
-    margin-top: 1px;
+    > .nx-chevron {
+      align-self: center;
+      color: #2c2c2c;
+      grid-area: actions;
+      justify-self: end;
+    }
   }
 }
 
-/*
- * Definition Lists
- *
- * Typically use <dl>
- */
-
-.nx-list--definition-list {
-  box-sizing: border-box;
+.nx-list-actions {
+  @include container-horizontal;
 
-  display: flex;
-  flex-wrap: wrap;
+  grid-area: actions;
```

Comment:
I think these should only be in the top-right grid area, not the entire right-hand side of the grid.  See in the attached screenshot how they are causing uneven vertical spacing around the list item text: 
![image](https://user-images.githubusercontent.com/3630091/93790337-eaefb900-fc00-11ea-9f84-8686a6806ef8.png)

Zeplin also currently shows the buttons being only 12px from the top and bottom of the list item, which presents a problem in terms of the current 16px padding.

Reply:
> I think these should only be in the top-right grid area, not the entire right-hand side of the grid. See in the attached screenshot how they are causing uneven vertical spacing around the list item text:
> ![image](https://user-images.githubusercontent.com/3630091/93790337-eaefb900-fc00-11ea-9f84-8686a6806ef8.png)
> 
> Zeplin also currently shows the buttons being only 12px from the top and bottom of the list item, which presents a problem in terms of the current 16px padding.

The issue there is that the buttons are taller than the text and so whether they span the rows, or make the top row taller the result is roughly the same.
<img width="991" alt="Screen Shot 2020-09-21 at 3 18 49 PM" src="https://user-images.githubusercontent.com/1991443/93811221-c950fa80-fc1d-11ea-880c-a489a7706308.png">

IMO a better solution would be to vertically centre the list text so that it lines up with the icon, but that effects the spacing between the list text and the sub-text and the list top
<img width="981" alt="Screen Shot 2020-09-21 at 3 22 24 PM" src="https://user-images.githubusercontent.com/1991443/93811540-47ad9c80-fc1e-11ea-9004-4aee04987a6a.png">

Reply:
> I think these should only be in the top-right grid area, not the entire right-hand side of the grid. See in the attached screenshot how they are causing uneven vertical spacing around the list item text:
> ![image](https://user-images.githubusercontent.com/3630091/93790337-eaefb900-fc00-11ea-9f84-8686a6806ef8.png)
> 
> Zeplin also currently shows the buttons being only 12px from the top and bottom of the list item, which presents a problem in terms of the current 16px padding.

Tweak for grid-area here, confines the buttons to row 1 column 2, no spanning: https://github.com/sonatype/sonatype-react-shared-components/pull/141/commits/58b3840f281bf7d46f5873dc5bba3e85d9f34176

Reply:
> > I think these should only be in the top-right grid area, not the entire right-hand side of the grid. See in the attached screenshot how they are causing uneven vertical spacing around the list item text:
> > ![image](https://user-images.githubusercontent.com/3630091/93790337-eaefb900-fc00-11ea-9f84-8686a6806ef8.png)
> > Zeplin also currently shows the buttons being only 12px from the top and bottom of the list item, which presents a problem in terms of the current 16px padding.
> 
> Tweak for grid-area here, confines the buttons to row 1 column 2, no spanning: [58b3840](https://github.com/sonatype/sonatype-react-shared-components/commit/58b3840f281bf7d46f5873dc5bba3e85d9f34176)

This moves the buttons up 4px: https://github.com/sonatype/sonatype-react-shared-components/pull/141/commits/97c3da2f2f514c64b1e203a9ff4cbfb294ac8324

Reply:
> This moves the buttons up 4px: 97c3da2

-4px bottom margin is also necessary; the buttons are still making the row 4px taller.

Reply:
> > This moves the buttons up 4px: [97c3da2](https://github.com/sonatype/sonatype-react-shared-components/commit/97c3da2f2f514c64b1e203a9ff4cbfb294ac8324)
> 
> -4px bottom margin is also necessary; the buttons are still making the row 4px taller.

https://github.com/sonatype/sonatype-react-shared-components/pull/141/commits/0a71927fdd75d5814f7c9b7f9cf8ee37648b7183